src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/BinaryExpression.cs

```diff
@@ -940,28 +989,58 @@ binaryType switch
+        /// <param name="left">An <see cref="System.Linq.Expressions.Expression" /> to set the <see cref="System.Linq.Expressions.BinaryExpression.Left" /> property equal to.</param>
+        /// <param name="right">An <see cref="System.Linq.Expressions.Expression" /> to set the <see cref="System.Linq.Expressions.BinaryExpression.Right" /> property equal to.</param>
+        /// <param name="liftToNull"><see langword="true" /> to set <see cref="System.Linq.Expressions.BinaryExpression.IsLiftedToNull" /> to <see langword="true" />; <see langword="false" /> to set <see cref="System.Linq.Expressions.BinaryExpression.IsLiftedToNull" /> to <see langword="false" />.</param>
+        /// <param name="method">A <see cref="System.Reflection.MethodInfo" /> to set the <see cref="System.Linq.Expressions.BinaryExpression.Method" /> property equal to.</param>
+        /// <returns>A <see cref="System.Linq.Expressions.BinaryExpression" /> that has the <see cref="System.Linq.Expressions.Expression.NodeType" /> property equal to <see cref="System.Linq.Expressions.ExpressionType.Equal" /> and the <see cref="System.Linq.Expressions.BinaryExpression.Left" />, <see cref="System.Linq.Expressions.BinaryExpression.Right" />, <see cref="System.Linq.Expressions.BinaryExpression.IsLiftedToNull" />, and <see cref="System.Linq.Expressions.BinaryExpression.Method" /> properties set to the specified values.</returns>
+        /// <exception cref="System.ArgumentNullException"><paramref name="left" /> or <paramref name="right" /> is <see langword="null" />.</exception>
+        /// <exception cref="System.ArgumentException"><paramref name="method" /> is not <see langword="null" /> and the method it represents returns <see langword="void" />, is not <see langword="static" /> (<see langword="Shared" /> in Visual Basic), or does not take exactly two arguments.</exception>
+        /// <exception cref="System.InvalidOperationException"><paramref name="method" /> is <see langword="null" /> and the equality operator is not defined for <paramref name="left" />.Type and <paramref name="right" />.Type.</exception>
+        /// <remarks>The resulting <see cref="System.Linq.Expressions.BinaryExpression" /> has the <see cref="System.Linq.Expressions.BinaryExpression.Method" /> property set to the implementing method. The <see cref="System.Linq.Expressions.Expression.Type" /> property is set to the type of the node. If the node is lifted, the <see cref="System.Linq.Expressions.BinaryExpression.IsLifted" /> property is <see langword="true" /> and the <see cref="System.Linq.Expressions.BinaryExpression.IsLiftedToNull" /> property is equal to <paramref name="liftToNull" />. Otherwise, they are both <see langword="false" />. The following information describes the implementing method, the node type, and whether a node is lifted.
```

Comment:
I think these headings and bullets [need to be converted away from markdown](https://github.com/carlossanlop/DocsPortingTool/blob/main/BackportInstructions.md#user-content-important:~:text=Check%20that%20there%20is%20no%20markdown%20in%20the%20plain%20xml%20sections).

It looks like there are going to be quite a few occurrences of this. I am only commenting on this one.

Reply:
Thanks, I had missed that document and wasn't aware of quite a few of those steps.